package.json

```diff
@@ -2,18 +2,22 @@
   "name": "the-graphql-workshop",
   "private": true,
   "type": "module",
-  "workspaces": ["src/*"],
+  "workspaces": [
+    "src/*"
+  ],
   "version": "1.0.0",
   "license": "CC-BY-SA-4.0",
   "author": "Sameer Srivastava <sameer.srivastava@nearform.com>",
-  "contributors": ["Simone Busoli <simone.busoli@nearform.com>"],
+  "contributors": [
+    "Simone Busoli <simone.busoli@nearform.com>"
+  ],
   "scripts": {
     "build": "slidev build",
     "start": "slidev --open",
     "export": "slidev export",
-    "db:up": "docker-compose up -d",
+    "db:up": "docker compose up -d",
     "db:migrate": "postgrator",
-    "db:down": "docker-compose down",
+    "db:down": "docker compose down",
```

Comment:
Updated docker-compose syntax, to align with the [fastify-workshop](https://github.com/nearform/the-fastify-workshop/blob/master/package.json#L14)

Reply:
yep good catch 👌